Content.Server/Administration/PlayerLocator.cs

```diff
@@ -118,7 +125,15 @@ internal sealed class PlayerLocator : IPlayerLocator
                 return null;
             }
 
-            return new LocatedPlayerData(userId, null, null);
+            var responseData = await resp.Content.ReadFromJsonAsync<UserDataResponse>(cancellationToken: cancel);
```

Comment:
This does not actually work. The HTTP request was sent as HEAD (because previously, this code path was only used as a does-exist check, so HEAD was sufficient) and no response body is actually returned. You need to change the HTTP method above to GET.

Also, please test your code next time :heck: